sunpy/map/sources/soho.py

```diff
@@ -65,8 +65,7 @@ def __init__(self, data, header, **kwargs):
         GenericMap.__init__(self, data, header, **kwargs)
 
         # Fill in some missing info
-        self.meta['detector'] = "EIT"
-        self.meta['waveunit'] = "Angstrom"
```

Comment:
`waveunit` should NOT be removed here. EIT channels are denoted by wavelengths and thus the `wavlnth` keyword needs a unit attached to it here